src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -207,9 +207,10 @@ private void setUpAlertsTriggers(List<Trigger> triggers) {
         List<Trigger> filteredTriggers = new ArrayList<>();
 
         for (Trigger trigger : triggers) {
-            if (trigger.getId().endsWith(resource.getId())) {
-                filteredTriggers.add(trigger);
-            }
+                if ( trigger.getTags()!=null && trigger.getTags().containsKey("resourceId") && trigger.getTags().get
```

Comment:
there may be tag or no tag and that tag may contain or may not contain resourceId so I thought to be on safe side